css/autotheme.css

```diff
@@ -0,0 +1,68 @@
+.autotheme {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+}
+
+.autotheme a {
+  display: flex;
+  flex: auto;
+  align-items: center;
+
+  color: var(--text-color);
+  font-style: normal;
+}
+
+.autotheme i {
+  width: unset;
+  height: 21px; /* same as lineheight */
+  margin-left: 1em;
+  align-self: center;
+}
+
+.autotheme-palette {
+  display: flex;
+  justify-content: space-around;
+
+  flex: auto;
+}
+
+body:not(.has-autotheme) .hide-when-no-autotheme {
```

Comment:
The class "has-autotheme" is set when the user selected a theme with autotheme. (yeah, seems obvious).
This makes it easy to just show and hide the autotheme-related elements.